html/index.hbs

```diff
@@ -1 +1,17 @@
-<div class="container"><h1>Homepage content here</h1></div>
\ No newline at end of file
+<div class="container mt-5">
+  <div class="media">
+    <div class="media-body">
+      <h1 class="mt-0">Learn how to write cleaner code with Yellow Brick Code</h1>
+      <h2 class="text-muted"></h2>
+      <p>Yellow Brick Code is a web app that aims to teach coders to write healthy and maintainable
+      code. Practice your coding skills by submitting your solution to a coding problem, then
+      see how your solution compares to others based on its complexity.</p>
+      <p>Our goal is to enable any programmer to learn how to write readable code like a software engineer;
```

Comment:
this is an awkward sentence, suggest re-phrasing

Reply:
done, ty!